docs/robot_playwright.rst

```diff
@@ -0,0 +1,164 @@
+=============================================
+Experimental Playwright Support
+=============================================
+
+Since its inception, CumulusCI has relied on Selenium to provide the
+foundation of our browser automation keywords.
+
+In 2020, Microsoft introduced a new browser automation tool named
+`Playwright <https://playwright.dev/>`_.
+Playwright is a ground-up reinvention of a browser automation
+library that aims to address several shortcomings of Selenium.  For
+example, playwright has built-in support for waiting for elements to
+appear, for working with the shadow DOM, video capture of a testing
+session, and so on.
+
+In 2021 the Robot Framework project introduced the
+`Browser <https://robotframework-browser.org/>`_ library which adds
+keywords based on top of Playwright.
```

Comment:
"Based on top of" rings strange to me. Are they keywords that control Playwright? Keywords that use the Playwright API? Keywords based on Playwright? Keywords from Playwright?

Reply:
How about _"...which adds keywords that use the Playwright API"_?

Background: Playwright itself is a library that knows nothing about keywords; Browser is the keyword library that uses the functions in the playwright library. 

I sure wish the robot folks had named the library PlaywriteLibrary rather than Browser. I think that might have made some of this less confusing.